incountry/crypto_utils.py

```diff
@@ -10,83 +13,91 @@
 
 
 KEYS_TO_HASH = SERVICE_KEYS + SEARCH_KEYS
+EXTRA_KEYS_TO_ENCRYPT = ["precommit_body"]
 
 KEYS_TO_OMIT_ON_ENCRYPTION = ["created_at", "updated_at"]
 
-RESERVED_KEYS_TO_SEND = ["version", "is_encrypted"]
+RESERVED_KEYS_TO_SEND = ["body", "version", "is_encrypted"]
 
 
-def validate_crypto(crypto):
+def validate_crypto(crypto: InCrypto):
     if not isinstance(crypto, InCrypto):
         raise StorageClientException(f"'crypto' argument should be an instance of InCrypto. Got {type(crypto)}")
 
 
-def validate_is_string(value, arg_name):
+def validate_is_string(value: str, arg_name: str):
     if not isinstance(value, str):
         raise StorageClientException(f"'{arg_name}' argument should be of type string. Got {type(value)}")
 
 
-def is_json(data):
+def is_json(data: str) -> bool:
     try:
         json.loads(data)
     except ValueError:
         return False
     return True
 
 
-def hash(data):
+def hash(data: str) -> str:
     return hashlib.sha256(data.encode("utf-8")).hexdigest()
 
 
-def normalize_key(key, normalize=False):
+def normalize_key(key: Any, normalize: bool = False) -> Any:
     if not isinstance(key, str):
         return key
     return key.lower() if normalize else key
 
 
-def normalize_key_value(value, normalize=False):
+def normalize_key_value(value: Any, normalize: bool = False) -> Any:
     if value is None:
         return None
     if isinstance(value, list):
         return [normalize_key(x, normalize=normalize) for x in value]
     return normalize_key(value, normalize=normalize)
 
 
-def get_salted_hash(value, salt):
+def get_salted_hash(value: str, salt: str) -> str:
     validate_is_string(value, "value")
     validate_is_string(salt, "salt")
     return hash(value + ":" + salt)
 
 
-def hash_string_key_value(value, salt, normalize=False):
+def hash_string_key_value(value: Union[str, list, None], salt: str, normalize=False) -> Union[str, list, None]:
```

Comment:
```suggestion
T = TypeVar("T", str, list, None)

def hash_string_key_value(value: T, salt: str, normalize=False) -> T:
```

By the way, you can name your type variables however you like but usually they just use `T` in docs.